docs/docs/03-light-clients/05-tendermint/01-overview.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Overview
+sidebar_label: Overview
+sidebar_position: 1
+slug: /ibc/light-clients/tendermint/overview
+---
+
+# `07-tendermint`
+
+## Overview
+
+The tendermint client is the first and most deployed light client in IBC. It implements the IBC Light Client interface to track a counterparty running Comet BFT consensus. Note: Tendermint is the old name of Comet BFT which has been retained in IBC to avoid expensive migration costs.
+
+## Synopsis
+
+### Initialization
+
+The Tendermint light client is initialized with a ClientState that contains parameters necessary for CometBFT header verification along with a latest height and ConsensusState which encapsulates a root of trust header that will serve to verify future incoming headers from the counterparty.
```

Comment:
```suggestion
The Tendermint light client is initialized with a ClientState that contains parameters necessary for CometBFT header verification along with a latest height and ConsensusState which encapsulates the application state root of a trust header that will serve to verify future incoming headers from the counterparty.
```

Reply:
Sorry I don't mean application root here, i mean the initial header itself serves as a root-of-trust. Will hyphenate for clarity